src/Sql/Sql/ManagedInstanceDnsAlias/Cmdlet/RemoveAzureSqlManagedInstanceDnsAlias.cs

```diff
@@ -0,0 +1,130 @@
+﻿using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.Sql.ManagedInstance.Model;
+using Microsoft.Azure.Commands.Sql.ManagedInstanceDnsAlias.Model;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Sql.ManagedInstanceDnsAlias.Cmdlet
+{
+    [Cmdlet(VerbsCommon.Remove, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlInstanceDnsAlias",
+        DefaultParameterSetName = DeleteByNameParameterSet,
+        SupportsShouldProcess = true),
+        OutputType(typeof(bool))]
```

Comment:
i think in your tests there's a comment like:
```
 #Since -PassThru was passed, we expect that the remove cmdlet returned the original object
 ```
 that sounds a bit contradictory to the specified `OutputType`, can you double check what's up?